templates/named.conf.j2

```diff
@@ -3,25 +3,9 @@ zone "{{ domain.name }}" {
    type master;
    file "/var/lib/bind/{{ domain.name }}.db";
    update-policy {
-       {% for subdomain in domain.subdomains %}
-       grant {{ subdomain.name }}. name {{ subdomain.name }}. A AAAA TXT MX CAA;
-       grant {{ subdomain.name }}. name *.{{ subdomain.name }}. A AAAA;
-       grant {{ subdomain.name }}. name mail._domainkey.{{ subdomain.name }}. TXT;
-       grant {{ subdomain.name }}. name _dmarc.{{ subdomain.name }}. TXT;
-       grant {{ subdomain.name }}. name _xmpp-client._tcp.{{ subdomain.name }}. SRV;
-       grant {{ subdomain.name }}. name _xmpp-server._tcp.{{ subdomain.name }}. SRV;
-       grant {{ subdomain.name }}. name xmpp-upload.{{ subdomain.name }}. A AAAA CNAME;
-       grant {{ subdomain.name }}. name muc.{{ subdomain.name }}. A AAAA CNAME;
-       grant {{ subdomain.name }}. name vjud.{{ subdomain.name }}. A AAAA CNAME;
-       grant {{ subdomain.name }}. name pubsub.{{ subdomain.name }}. A AAAA CNAME;
-       {% endfor %}
+       grant * selfsub . A AAAA TXT MX CAA CNAME SRV DNAME;
```

Comment:
See https://bind9.readthedocs.io/en/v9.18.31/reference.html#namedconf-statement-update-policy

If i understand well, all identity matched by * (referring to a tsig key containing the domain name) could edit domain and subdomain related to the name the key...